Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1912,6 +1899,15 @@ PhysicalParticleContainer::PushPX (WarpXParIter& pti,
                        t_chi_max,
 #endif
                        dt);
+
+#ifdef WARPX_QED
+    if (local_has_quantum_sync) {
+        evolve_opt(ux[ip], uy[ip], uz[ip],
+                   Exp, Eyp, Ezp,Bxp, Byp, Bzp,
```

Comment:
```suggestion
                   Exp, Eyp, Ezp, Bxp, Byp, Bzp,
```
That was frustrating :D

Reply:
:rofl: